pkg/addon/policyframework/manifests/managedclusterchart/templates/namespace.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright Contributors to the Open Cluster Management project
+
+{{- if not .Values.onMulticlusterHub }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    policy.open-cluster-management.io/isClusterNamespace: "true"
```

Comment:
I don't think this label is being used anymore. It was added for compliance operator integration but no longer needed anymore. Can you remove it?

Reply:
Done, thanks for noticing that